### PR TITLE
Add Snapshot Testing to Gallery

### PR DESCRIPTION
**Pull request by chiaramooney**
### Why
Add Snapshot Testing to Gallery

Snapshot tests for all component pages now added.

### CODE REVIEW

File: __tests__/App-test.js [hunk 1]
```diff
@@ -1,14 +1,215 @@
...
+import {PressableExamplePage} from '../src/examples/PressableExamplePage';
+import {FlatListExamplePage} from '../src/examples/FlatListExamplePage';
+import {ScrollViewExamplePage} from '../src/examples/ScrollViewExample';
+import {PopupExamplePage} from '../src/examples/PopupExamplePage';
+import {FlyoutExamplePage} from '../src/examples/FlyoutExamplePage';
+import {CheckBoxExamplePage} from '../src/examples/CheckBoxExamplePage';
+import {ConfigExamplePage} from '../src/examples/ConfigExamplePage';
+import {DatePickerExamplePage} from '../src/examples/DatePickerExamplePage';
+import {TimePickerExamplePage} from '../src/examples/TimePickerExamplePage';
+//import {SketchExamplePage} from '../src/examples/SketchExamplePage';
```
Comment (AgneLukoseviciute) [inline __tests__/App-test.js:20]: for the sake of leaving some breadcrumbs, do you know what the issue was with including this and track player?

File: jest-setup.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-undef */
+import {NativeModules} from 'react-native';
+jest.mock('react-native-permissions', () =>
```
Comment (AgneLukoseviciute) [inline jest-setup.js:3]: do we still need this since we're excluding the permissions page?

File: __tests__/App-test.js [hunk 3]
```diff
@@ -1,14 +1,215 @@
...
+
+test('Expander Example Page', async () => {
+  let tree;
+  await act(() => {
+    tree = create(<ExpanderExamplePage />);
+  });
+  expect(tree.toJSON()).toMatchSnapshot();
+});
+
+/*test('Permissions Example Page', async () => {
```
Comment (AgneLukoseviciute) [inline __tests__/App-test.js:83]: same as above for this. + maybe we could add something to the readme or wiki on how to update the snapshots and why they might fail.

Comment (AgneLukoseviciute) [reply __tests__/App-test.js:83]: Or even just link to the main repo snapshot documentation somewhere.

**Review by AgneLukoseviciute — APPROVED** [review]

File: __tests__/App-test.js (hunk 1 above)
Comment (chiaramooney) [reply __tests__/App-test.js:20]: Yes I can add comments!

File: jest-setup.js (hunk 2 above)
Comment (chiaramooney) [reply jest-setup.js:3]: Leaving in because we will need it once I figure out the permissions page issue!

File: __tests__/App-test.js (hunk 3 above)
Comment (chiaramooney) [reply __tests__/App-test.js:83]: Will add comments

**Pull request merged**